device_plugins/deploy_qat.md

```diff
@@ -1,4 +1,19 @@
 # Create Intel QAT Device Plugin CR
+Intel® QuickAssist Technology (Intel® QAT) provides cryptographic and compression acceleration capabilities used to improve performance and efficiency across the data center.
+For more details see https://intel.github.io/quickassist
+
+## Prerequisite
+Before deploying the QAT plugin, make sure iommu MCO is deployed and iommu is enabled.
+By default, half of the virtual functions are used for compression and the other half is used for cryptography. This default behaviour can be customised via a config map which configures the hardware using sysfs. The config values could be `asym`, `sym` and `dc`, for more details see [cfg_services](https://github.com/torvalds/linux/blob/42e66b1cc3a070671001f8a1e933a80818a192bf/Documentation/ABI/testing/sysfs-driver-qat)
```

Comment:
This should not be the Prerequisite. I think it should be the feature Gap, I suggest to have an individual section in this document to introduce customers to the detail. And the link to this issue should be added in the section. And above link is good for the stand-alone RHEL server. But for OCP, the ender user should not directly operate the individual node.

Reply:
> This should not be the Prerequisite. I think it should be the feature Gap,

Which part you are referring to?